L/libjulia/common.jl

```diff
@@ -158,7 +164,7 @@ function configure(version)
     rm -rf /workspace/srcdir/julia-1.5.1/deps/checksums/lapack-3.9.0.tgz
 
     # compile libjulia but don't try to build a sysimage
-    make USE_CROSS_FLISP=1 NO_GIT=1 LDFLAGS=${LDFLAGS} CFLAGS=${CFLAGS} CXXFLAGS=${CXXFLAGS} -j${nproc} VERBOSE=1 julia-ui-release
+    make USE_CROSS_FLISP=1 NO_GIT=1 LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}" -j${nproc} VERBOSE=1 julia-ui-release
```

Comment:
Heads-up that `julia-ui-release` is called `julia-cli-release` on 1.6+, so you may want to parameterize that.

Reply:
Noted, thanks